framework/package-lock.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "lisk-service-framework",
-  "version": "1.4.4",
+  "version": "1.4.5",
```

Comment:
Also, kindly run `npm install` so that all the sub-dependencies are up-to-date as well.